### DISCUSSION TITLE
Is it possible to disable overlay ?

### DISCUSSION THREAD

**Discussion by polc** [parcel-bundler/parcel#8830] [Q&A]
Hi ! Do you know if it's possible to disable the default overlay ? When we trigger some HTTP requests that do not resolve, it show the overlay, but in our app it often means that some services are not started, which is OK in our dev environment. 

We use Suspense in React, these requests are catched by error boundaries (https://reactjs.org/docs/error-boundaries.html), but still trigger the overlay.

Thanks !

**Comment by Cristy94** [comment]
I have a similar issue, where I can't see the errors while testing React Error Boundary, because I see the Parcel error overlay instead.

EDIT: I see this one, to be more specific: https://parceljs.org/blog/v2-6-0/#react-error-overlay
